go-controller/pkg/ovn/multicast.go

```diff
@@ -20,6 +20,13 @@ const (
 	ipv6DynamicMulticastMatch = "(ip6.dst[120..127] == 0xff && ip6.dst[116] == 1)"
 	// Legacy multicastDefaultDeny port group removed by commit 40a90f0
 	legacyMulticastDefaultDenyPortGroup = "mcastPortGroupDeny"
+	// Suffixes used by Multicast ACL Names.
+	multicastAllowEgress         = "MulticastAllowEgress"
+	multicastAllowIngress        = "MulticastAllowIngress"
+	defaultDenyMulticastEgress   = "DefaultDenyMulticastEgress"
+	defaultDenyMulticastIngress  = "DefaultDenyMulticastIngress"
+	DefaultAllowMulticastEgress  = "DefaultAllowMulticastEgress"
+	DefaultAllowMulticastIngress = "DefaultAllowMulticastIngress"
```

Comment:
I guess these changes are not really needed in case we won't backport this pr, because in master branch they will be overwritten with the next acl indexes pr, wdyt?

Reply:
yes agree but now its done already lol let's leave it as is :) its ok if aclindexes is removing this.